src/pages/vault/downloads/index.tsx

```diff
@@ -3,25 +3,33 @@ import { GetStaticProps } from 'next'
 import vaultData from 'data/vault.json'
 import installData from 'data/vault-install.json'
 import { ProductData } from 'types/products'
-import { generateStaticProps, GeneratedProps } from 'lib/fetch-release-data'
 import CoreDevDotLayout from 'layouts/core-dev-dot-layout'
 import ProductDownloadsView from 'views/product-downloads-view'
+import { generateStaticProps } from 'views/product-downloads-view/server'
+import {
+  ProductDownloadsViewStaticProps,
+  RawProductDownloadsViewContent,
+} from 'views/product-downloads-view/types'
 
-const VaultDownloadsPage = (props: GeneratedProps): ReactElement => {
-  const { latestVersion, releases } = props
+const VaultDownloadsPage = ({
+  latestVersion,
+  releases,
+  pageContent,
+}: ProductDownloadsViewStaticProps): ReactElement => {
   return (
     <ProductDownloadsView
       latestVersion={latestVersion}
-      pageContent={installData}
+      pageContent={pageContent}
       releases={releases}
     />
   )
 }
 
 export const getStaticProps: GetStaticProps = async () => {
-  const product = vaultData as ProductData
-
-  return generateStaticProps(product)
+  return await generateStaticProps(
+    vaultData as ProductData,
+    installData as unknown as RawProductDownloadsViewContent
+  )
```

Comment:
Rather than passing `installData` directly to the view component, we now do some server-side stuff first - namely, fetching the tutorial and collection data based on authored slugs.

Reply:
Good call! I think that's the right approach.

🙋🏻 Why does `installData` need to be first cast as `unknown` before `RawProductDownloadsViewContent`? It's not needed for `vaultData`+ `ProductData`, so it's not clear to me yet why it's needed for `installData` + `RawProductDownloadsViewContent`.

Reply:
Ah good callout, I meant to ask about this.

I'm not sure why it's needed, but TypeScript gets real mad if I don't do this:

<img width="791" alt="CleanShot 2022-05-31 at 13 38 03@2x" src="https://user-images.githubusercontent.com/4624598/171248369-5c8c2b6b-43ec-4fed-8623-97c47adaf3e5.png">

I figured we must be setting up types for this data somewhere? But couldn't find anything. So I'm unsure why casting to unknown is needed.

Reply:
(sorry didn't mean to resolve!)